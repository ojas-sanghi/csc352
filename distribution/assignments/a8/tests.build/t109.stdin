6 26 
17 

24 15 

9 12 
27 28 


