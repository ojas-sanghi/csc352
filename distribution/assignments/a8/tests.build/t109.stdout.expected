(-6((((-9(-12-))15-)17(-24-))26(-27(-28-))))
Insert 10
(-6((((-9((-10-)12-))15-)17(-24-))26(-27(-28-))))
